### PR TITLE
Move playwright tests to this repo

### PR DESCRIPTION
**Pull request by wmannard**
## [JSUI-3542](https://coveord.atlassian.net/browse/JSUI-3542)

Moves the JSUI Playwright tests to this repo

- They will be closer to the code they test
- It will be easier to use them as a release certifier

(This is a public repo, so it can’t use Coveo's Github runners. This means that the Github certifier doesn't to have access to the private `sfint-testauto` repo out of the box)

Git history should be preserved
Added the QA team as owners for this directory

---

**Note: This is move from a private repo to a public one, so keep an eye out for anything that shouldn't be exposed.**

---

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

[JSUI-3542]: https://coveord.atlassian.net/browse/JSUI-3542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -7,3 +7,6 @@
 # Prevent PRs that only modify dependencies from notifying everyone
 package.json @ghost
 yarn.lock @ghost
+
+# Owners for the Playwright end-to-end test directory
+playwright/ @coveo/search-qa
```
Comment (wmannard) [inline CODEOWNERS:12]: I get the error "Unknown owner: make sure the team @coveo-platform/qa exists, is publicly visible, and has write access to the repository"
...but this looks right: https://github.com/orgs/coveo/teams/search-qa

Comment (wmannard) [inline CODEOWNERS:12]:
```suggestion
playwright/ @wmannard @lvu285 
```

Comment (louis-bompart) [reply CODEOWNERS:12]: No need for suggestion 👍

Comment (louis-bompart) [reply CODEOWNERS:12]: Write access granted!

**Review by louis-bompart — APPROVED** [review]

**Review by alexprudhomme — APPROVED** [review]

**Pull request merged**